src/linker/utilities/general_utils.py

```diff
@@ -103,3 +103,10 @@ def exit_with_validation_error(error_msg: Dict) -> None:
         "\n==========================================\n"
     )
     exit(errno.EINVAL)
+
+
+def merge_dicts(dicts):
+    final_dict = {}
+    for dictionary in dicts:
```

Comment:
Let's golf: `return {key: val for dict in dicts for key, val in dict.items()}`

Reply:
very nice! Let's see Paul Allen's dict util